assignment_0/a0-pipeline.py

```diff
@@ -0,0 +1,34 @@
+import subprocess
+
+container_name = "assignment-0"
+
+
+def docker_build():
+    try:
+        # Run 'docker build' command to build the Docker image from the current directory
+        subprocess.run(["docker", "build", "-t", container_name, "."], check=True)
+        print("Docker build successful.")
+    except subprocess.CalledProcessError as e:
+        print("Error: Docker build failed.")
+        print(e)
+
+
+def docker_run():
+    try:
+        # Run 'docker run' command to run the built Docker image
+        subprocess.run(
+            ["docker", "run","--name", container_name, "-p", "5001:5000", "-d", container_name], check=True
```

Comment:
Assignment 0 and 1 now both use TCP port 5001. Maybe change it to a different one? Or you can choose to make the stop and remove container part of this assignment.